examples/apps/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
 
 INCLUDE(CMakeVariables.txt)
+SET(GENERIC_PRIVATE_INCLUDE_DIRS "." "${TCF_TOP_DIR}/tc/sgx/common"
+	"${TCF_TOP_DIR}/tc/sgx/common/work_order_interface")
```

Comment:
What's the dependency with ${TCF_TOP_DIR}/tc/sgx/common"

Reply:
work_order_interface includes error.h from ${TCF_TOP_DIR}/tc/sgx/common.